### ISSUE TITLE
Route automatically redraw when a shortest path is found

### ISSUE THREAD

**Issue by leonardoayres** [flutter/flutter#57564]
I'm working on a route record app module using **Google Maps Flutter** with **Locator** package to track the current GPS position. To draw the route, i'm using the **google_map_polyline** package.

When i'm run, the polyline is drawn on the map without issues (always using a single polyline), but if my route gets near to the starting point, the map automacally ignore the previous route redrawing it to the shortest path to the starting point.

TL; DR;
I can never do a circular route because the map always deduce the shortest path for me when i'm getting close to the start.

Is that an issue of some configuration that was not applied or a package issue?

**Comment by TahaTesser** [comment]
Hi @leonardoayres 
can you please provide your `flutter doctor -v` and `flutter run --verbose`? 
Also, to better address the issue, would be helpful if you could post a minimal code sample to reproduce the problem
Thank you
